Source/Dafny/DafnyAst.cs

```diff
@@ -1839,17 +1839,17 @@ public enum AutoInitInfo { MaybeEmpty, Nonempty, CompilableValue }
     /// really a meet, so the caller should set up additional constraints that the result is
     /// assignable to both a and b.
     /// </summary>
-    public static Type Meet(Type a, Type b, BuiltIns builtIns) {
+    public static Type Join(Type a, Type b, BuiltIns builtIns) {
```

Comment:
Please also make the change in the comment for this method.

Reply:
Several comments in the Dafny source code mentions either "meet" or "join". Those also need changing.

Reply:
In a similar way, does the comment/description/behavior of `ComputeExtrema` need to change?